start-automaker.sh

```
@@ -9,13 +9,29 @@ set -e
 # ============================================================================
 # CONFIGURATION & CONSTANTS
 # ============================================================================
+APP_NAME="Automaker"
```

Shellcheck (0.11.0)

[warning] APP_NAME appears unused. Verify use (or export if used externally).

(SC2034)